.github/workflows/run-nethermind-tests.yml

```diff
@@ -267,10 +267,10 @@ jobs:
         dotnet test -c Release src/Nethermind/Ethereum.KeyAddress.Test
     - name: Ethereum.PoW.Test
       run: |
-        dotnet test -c Release src/Nethermind/Ethereum.Rlp.Test
+        dotnet test -c Release src/Nethermind/Ethereum.PoW.Test
```

Comment:
Basic are now PoW tests?

Reply:
Nope. Basic are basic (lines 252-254) and was second time as Rlp (line 273). And Rlp was as PoW. Now basic is basic, Rlp is Rlp and PoW is PoW